addons/controllers/clusterbootstrap_controller_test.go

```diff
@@ -170,26 +170,28 @@ var _ = Describe("ClusterBootstrap Reconciler", func() {
 				Expect(hasPackageInstalls(ctx, k8sClient, cluster, constants.TKGSystemNS,
 					clusterBootstrap.Spec.AdditionalPackages, logr.Logger{})).To(BeTrue())
 
-				By("packageinstalls for additional packages should all have the correct owner reference set")
-				var ownerIsCluster bool
+				By("packageinstalls for core packages should not have owner references")
+				var corePackages []*runtanzuv1alpha3.ClusterBootstrapPackage
+				corePackages = append(corePackages, clusterBootstrap.Spec.CNI, clusterBootstrap.Spec.CPI, clusterBootstrap.Spec.CSI)
```

Comment:
It would be nice to be able to pull all the PI for a cluster. 
We can do it for additional package, but for core packages the list is hardcoded here: 
https://github.com/vmware-tanzu/tanzu-framework/blob/8e84c7a002814a6711610e11481c653ba48eb770/addons/controllers/clusterbootstrap_controller.go#L241